packages/shadydom/src/shadydom.js

```diff
@@ -118,6 +118,81 @@ if (utils.settings.inUse) {
     'nativeMethods': nativeMethods,
     'nativeTree': nativeTree,
     'patchElementProto': patchElementProto,
+    // Use this setting to choose the implementation of `querySelector` and
+    // `querySelectorAll`. The logical tree exposed by Shady DOM via the DOM
```

Comment:
Maybe add: "The options provide different trade offs between accuracy and performance, as explained below."

Reply:
I added that sentence and reworded this paragraph a bit.